google/cloud/bigtable/examples/run_examples_emulator.sh

```diff
@@ -31,16 +31,17 @@ readonly ZONE_ID="fake-zone"
 
 # The examples are noisy,
 for example in instance_admin table_admin data; do
-  LOG=$(mktemp --tmpdir "bigtable_examples_${example}_XXXXXXXXXX.log")
+  log=$(mktemp --tmpdir "bigtable_examples_${example}_XXXXXXXXXX.log")
   echo "${COLOR_GREEN}[ RUN      ]${COLOR_RESET} ${example}"
-  run_all_${example}_examples "${PROJECT_ID}" "${ZONE_ID}" >${LOG} 2>&1 </dev/null
+  run_all_${example}_examples "${PROJECT_ID}" "${ZONE_ID}" # >${log} 2>&1 </dev/null
```

Comment:
This is a new comment character; you are losing all logs here. Was this change intended?

Reply:
Nope, that was a mistake. Fixed, and thanks!